README.md

```diff
@@ -1,64 +1,97 @@
 
-To be on the safe side, it is still recommended to make a complete backup of the device flash __**before**__ running the installer!
+**WARNING #1** This will replace the bootloader (TF-A 2.4, U-Boot 2021.10) and convert the flash layout of the device to UBI! The installer stores a copy of the previous bootchain in a dedicated UBI volume `boot_backup`.
 
-To use the script to generate the installer image, you will need all runtime requirements of the OpenWrt ImageBuilder installed as well as `libfdt-dev` and `cmake` which is needed by the installer-generator itself.
+**WARNING #2** Re-flashing the installer when the device is already using UBI flash layout will erase the previously backed up bootchain, which in most cases would be the vendor/official one.
 
-You may of course as well go ahead and download the generated files [here](https://github.com/dangowrt/linksys-e8450-openwrt-installer/releases).
+For utmost safety (but not absolutely necessary), you are recommended to make a complete backup of the device flash __**before**__ running the installer. (see below "Device Flash Backup Procedure")
 
-## Steps
-#### (assuming the device runs stock firmware and is new or just after factory reset)
+You'll need the below to use the script to generate the installer image:
+* OpenWrt ImageBuilder
+* `libfdt-dev`
+* `cmake`
 
-1. Connect any of the LAN ports of the device directly to the Ethernet port of your PC.
-2. Configure the IP address of the PC to be `192.168.1.254`, netmask `255.255.255.0`, no gateway, no DNS.
+If you are not interested in building yourself, the pre-built files are available [here](https://github.com/dangowrt/linksys-e8450-openwrt-installer/releases).
+
+## Installing OpenWrt
+#### Assuming the device is running stock firmware, and is brand new or just after factory reset.
+
+1. Connect any of the LAN ports of the device directly to the Ethernet port of your computer.
+2. Set the IP address of your computer as `192.168.1.254` with netmask `255.255.255.0`, no gateway, no DNS.
 3. Power on the device, wait about a minute for it to be ready.
-4. Open a web browser and navigate to http://192.168.1.1 and wait for the wizard to come up.
-5. Click *exactly* inside the radio button to confirm you have read the terms and conditions, then abort the wizard.
-6. You should then be greeted by the login screen, the stock password is "admin". (This step might only apply to the Linksys, everything else remains the same) 
+4. Open a web browser, navigate to http://192.168.1.1 and wait for the wizard to come up.
+5. Click *exactly* inside the radio button to confirm the terms and conditions, then abort the wizard.
+6. You should then be greeted by the login screen, the stock password is "admin". (This step may be required for Linksys E8450 only)
```

Comment:
```suggestion
6. You should then be greeted by the login screen, the stock password is "admin".
```

I have personally verified this and yes, this is necessary on both, the Linksys E8450 and Belkin RT3200.